e2e-tests/micronaut-patio-api-version/target/classes/META-INF/services/io.micronaut.inject.BeanDefinitionReference

```diff
@@ -0,0 +1 @@
+com.foo.micronaut.patio.$IndexControllerDefinitionClass
```

Comment:
files under `target` must never be added under Git.
actually, each time a new module is created, its `target` folder must be added to `.gitignore`